tooling/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <VsixVersion>15.5</VsixVersion>
     <VsixVersion Condition="'$(BuildNumber)'!=''">$(VsixVersion).$(BuildNumber)</VsixVersion>
+    <VsixVersion Condition="'$(BuildNumber)'==''">$(VsixVersion).999999</VsixVersion>
```

Comment:
I was running into an issue building/running the VSIX. Since I have 15.5.<something> installed, that's a higher number than 15.5. Using 15.5.999999999999999 means that a local build is always 'higher' or at least the same.